### PR TITLE
[BrokenLinksH2] Fixed broken links

### PR DESCRIPTION
**Pull request by kturetz**
Fixed broken links in iis-docs.

### CODE REVIEW

**Comment by kturetz** [comment]
**Global effort to fix broken links** 

@wadepickett 
@Rick-Anderson 
@walterov   

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. I’d very much appreciate your timely review and feedback on the link changes I’ve suggested. Thanks!

File: iis/web-hosting/planning-the-web-hosting-architecture/sustainable-computing.md [hunk 1]
```diff
@@ -19,18 +19,18 @@ This article provides a map to the various sources of information on Sustainable
...
-- [Five Ways to Reduce Data Center Server Power Consumption](http://www.thegreengrid.org/gg_content/White_Paper_7_-_Five_Ways_to_Save_Power.pdf "Five Ways to Reduce Data Center Server Power Consumption")
+- Energy Efficiency Best Practices in Microsoft Data Center Operations (`https://www.microsoft.com/environment/our_commitment/articles/datacenter_bp.aspx "Energy Efficiency Best Practices in Microsoft Data Center Operations"`)
+- [Guidelines for Energy-Efficient Data Centers](https://community.exchange.se.com/t5/APC-Enterprise-Power-UPS-Forum/Green-Grid-White-Paper-Guidelines-for-Energy-Efficient-Data/m-p/282581#M910 "Guidelines for Energy-Efficient Data Centers")
+- [The Green Grid Data Center Power Efficiency Metrics: PUE and DCiE](https://www.missioncriticalmagazine.com/ext/resources/MC/Home/Files/PDFs/TGG_Data_Center_Power_Efficiency_Metrics_PUE_and_DCiE.pdf "The Green Grid Data Center Power Efficiency Metrics: PUE and DCiE")
+- [The Green Grid Peer Review of "DC Power for Improved Data Center Efficiency"](https://www.thegreengrid.org/en/resources/library-and-tools/243-WP)
```
Comment (wadepickett) [inline iis/web-hosting/planning-the-web-hosting-architecture/sustainable-computing.md:29]:
```suggestion
- [The Green Grid Peer Review of "DC Power for Improved Data Center Efficiency"](https://www.thegreengrid.org/resources/library-and-tools/243-WP)
```

File: iis/web-hosting/installing-infrastructure-components/system-center-solutions-vmm-amp-scom.md [hunk 2]
```diff
@@ -47,11 +47,11 @@ This video &lt;link: [https://mediadl.microsoft.com/mediadl/IISNET/Media/HDA20-1
...
+System Center Operations Manager 2007 &lt;link: [https://www.microsoft.com/systemcenter/operationsmanager/en/us/default.aspx](https://www.microsoft.com/systemcenter/operationsmanager/en/us/default.aspx) &gt; is the next step in the evolution of Microsoft operational event and performance monitoring solutions. This paper &lt;link: `https://download.microsoft.com/download/A/2/F/A2F199C0-672E-44E6-BF1D-878E233C3F08/OperationsManager%202007.zip` &gt; provides an overview of the features and functionality included in System Center Operations Manager 2007.
```
Comment (wadepickett) [inline iis/web-hosting/installing-infrastructure-components/system-center-solutions-vmm-amp-scom.md:54]:
```suggestion
System Center Operations Manager 2007 &lt;link: [https://www.microsoft.com/systemcenter/operationsmanager/default.aspx](https://www.microsoft.com/systemcenter/operationsmanager/default.aspx) &gt; is the next step in the evolution of Microsoft operational event and performance monitoring solutions. This paper &lt;link: `https://download.microsoft.com/download/A/2/F/A2F199C0-672E-44E6-BF1D-878E233C3F08/OperationsManager%202007.zip` &gt; provides an overview of the features and functionality included in System Center Operations Manager 2007.  
```

Comment (wadepickett) [inline iis/web-hosting/installing-infrastructure-components/system-center-solutions-vmm-amp-scom.md:54]:
```suggestion
System Center Operations Manager 2007 &lt;link: [https://www.microsoft.com/systemcenter/operationsmanager/default.aspx](https://www.microsoft.com/systemcenter/operationsmanager/default.aspx) &gt; is the next step in the evolution of Microsoft operational event and performance monitoring solutions. This paper &lt;link: `https://download.microsoft.com/download/A/2/F/A2F199C0-672E-44E6-BF1D-878E233C3F08/OperationsManager%202007.zip` &gt; provides an overview of the features and functionality included in System Center Operations Manager 2007.  
```

**Review by wadepickett — COMMENTED** [review]
Removing a few en\us and also forcing a new build since the original build timed out for some reason.

**Comment by wadepickett** [comment]
Still can't get a build.  Closing and re-opening this PR to force a new build...

**Review by wadepickett — APPROVED** [review]
Build was not running earlier due to some server timeouts.   Running now. Reviewed warnings, and removed a few en-us  from links.

**Pull request merged**